### PR TITLE
New influxd commands, flags, and config options

### LINKED ISSUE #2079: Add new influxd commands, flags, and config options
- [x] Add `--overwrite-existing-v2` flag to `influxd upgrade` (https://github.com/influxdata/influxdb/pull/20473)
- [x] Add `influxd print-config` command (https://github.com/influxdata/influxdb/pull/20524)
- [x] Add `nats-port` configuration option to `influxd` and related configuration documentation (https://github.com/influxdata/influxdb/pull/20561)
- [x] Add `nats-max-payload-bytes` configuration option to `influxd` and related configuration documentation (https://github.com/influxdata/influxdb/pull/20564)
- [x] Add `influxd inspect export-lp` command (https://docs.influxdata.com/influxdb/v2.0/reference/cli/influxd/)

##### Relevant URLs
- https://docs.influxdata.com/influxdb/v2.0/reference/cli/influxd/
- https://docs.influxdata.com/influxdb/v2.0/reference/cli/influxd/upgrade/
- https://docs.influxdata.com/influxdb/v2.0/reference/config-options/ (Config options doc)
- https://docs.influxdata.com/influxdb/v2.0/upgrade/v1-to-v2/

### PR DESCRIPTION
**Pull request by sanderson**
Closes #2079 

- [x] Tests pass (no build errors)
- [x] Rebased/mergeable

### CODE REVIEW

File: content/influxdb/v2.0/reference/cli/influxd/inspect/export-lp.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+influxd inspect export-lp [flags]
+```
+
+## Flags
+| Flag |                 | Description                                         | Input type |
+|:---- |:---             |:-----------                                         |:----------:|
+|      | `--bucket-id`   | ({{< req >}}) Bucket ID                             | string     |
+|      | `--compress`    | Compress output with GZIP                           |            |
+|      | `--end`         | End time to export (RFC3339 format)                 | string     |
+|      | `--engine-path` | Path to persistent InfluxDB engine files            | string     |
```
Comment (danxmoran) [inline content/influxdb/v2.0/reference/cli/influxd/inspect/export-lp.md:26]: Also required

File: content/influxdb/v2.0/reference/cli/influxd/inspect/export-lp.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+| `-h` | `--help`        | Help for the `export-lp` command.                   |            |
+|      | `--log-level`   | Log-level (`debug`, `info` _(default)_, or `error`) | string     |
+|      | `--measurement` | Measurement name(s) to export                       | strings    |
+|      | `--output-path` | Output file path                                    | string     |
```
Comment (danxmoran) [inline content/influxdb/v2.0/reference/cli/influxd/inspect/export-lp.md:30]: Also required

File: content/influxdb/v2.0/reference/cli/influxd/upgrade.md [hunk 3]
```diff
@@ -49,6 +49,7 @@ influxd upgrade [command]
 |      | `--log-level`                   | Supported log levels are `debug`, `info`, `warn`, and `error` (default `info`)                             | string     |
 |      | `--log-path`                    | Custom log file path (default `~/upgrade.log`)                                                             | string     |
 | `-o` | `--org`                         | Primary organization name                                                                                  | string     |
+|      | `--overwrite-existing-v2`       | Overwrite existing files at the `engine-path` instead of aborting the upgrade process                      |            |
```
Comment (danxmoran) [inline content/influxdb/v2.0/reference/cli/influxd/upgrade.md:52]: More than just the `engine-path`. The full list is:
* `bolt-path`
* `continuous-query-export-path`
* `influx-configs-path`
* `engine-path`
* `v2-config-path`

File: content/influxdb/v2.0/reference/cli/influxd/upgrade.md [hunk 4]
```diff
@@ -49,6 +49,7 @@ influxd upgrade [command]
 |      | `--log-level`                   | Supported log levels are `debug`, `info`, `warn`, and `error` (default `info`)                             | string     |
```
Comment (danxmoran) [inline content/influxdb/v2.0/reference/cli/influxd/upgrade.md:49]: Noticed that the `---influx-configs-path` option is missing. The text from `influxd upgrade -h` is:
```
-c, --influx-configs-path string            path for 2.x CLI configurations file (default "~/.influxdbv2/configs")
```

Comment (danxmoran) [inline content/influxdb/v2.0/reference/cli/influxd/upgrade.md:49]: Can't comment on it directly, but `--influx-command-path` above is no longer a supported CLI option.

**Review by noramullen1 — APPROVED** [review]

**Pull request merged**